core/src/com/unciv/ui/options/OptionsPopup.kt

```diff
@@ -147,6 +147,7 @@ class OptionsPopup(
                 }
             }
             withGLContext {
+                BaseScreen.setSkin()
```

Comment:
Why do we need this?

Reply:
A lot of UI elements rely on BaseScreen.skin. When changing skins, these UI elements do not get updated if setSkin is not called and would only update on a restart.